### PR TITLE
Update the project links on the community page 

### PR DESCRIPTION
**Pull request by yonaymoris**
**What kind of change does this PR introduce?**

Updated the dead links on the community page and added one more project with physical computing + ml5 — I thought it'd be useful to show ml5 applied outside the browser window, but let me know if it's not the right project for this!

### CODE REVIEW

File: lambda/hello.js [hunk 1]
```diff
@@ -1 +1 @@
...
... in r)e[t]=r[t]}(exports,function(e){var r={};function t(n){if(r[n])return r[n].exports;var o=r[n]={i:n,l:!1,exports:{}};return e[n].call(o.exports,o,o.exports,t),o.l=!0,o.exports}return t.m=e,t.c=r,t.d=function(e,r,n){t.o(e,r)||Object.defineProperty(e,r,{enumerable:!0,get:n})},t.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},t.t=function(e,r){if(1&r&&(e=t(e)),8&r)return e;if(4&r&&"object"==typeof e&&e&&e.__esModule)return e;var n=Object.create(null);if(t.r(n),Object.defineProperty(n,"default",{enumerable:!0,value:e}),2&r&&"string"!=typeof e)for(var o in e)t.d(n,o,function(r){return e[r]}.bind(null,o));return n},t.n=function(e){var r=e&&e.__esModule?function(){return e.default}:function(){return e};return t.d(r,"a",r),r},t.o=function(e,r){return Object.prototype.hasOwnProperty.call(e,r)},t.p="",t(t.s=0)}([function(e,r,t){"use strict";function n(e,r,t){console.log("queryStringParameters",e.queryStringParameters),t(null,{statusCode:200,body:JSON.stringify({msg:"Hello, World! "+Math.round(10*Math.random())})})}t.r(r),t.d(r,"handler",(function(){return n}))}]));
```
Comment (bomanimc) [inline lambda/hello.js:1]: Curious to know what caused the change here, but this looks like a file we can delete anyways! I'm happy to clean up this file later on.

**Review by bomanimc — CHANGES_REQUESTED** [review]
Hi @yonaymoris! Thank you for finding and correcting these links! It's great to correct these incorrect links.

I'm requesting changes on the PR because we typically request that new projects for the Community page are submitted through [the Google form linked on the Community page](https://docs.google.com/forms/d/e/1FAIpQLSdPz0ICzTSVdLAteIKwJ-zFzX6dX5l3dOpjWGzm6LIZutKvlA/viewform) so that other members of our team can take a look before it's published on the live site.

Would you mind separating the new project addition from this PR and placing it into a separate PR that we can merge if the new project you're suggesting for the Community page is approved?

cc: @c-dacanay for any further notes of the editorial process for Community page projects.

File: lambda/hello.js (hunk 1 above)
Comment (yonaymoris) [reply lambda/hello.js:1]: Got it, Just pushed another commit! 
Also, seems like it wrapped the last anonymous function into brackets in the hello.js, which caused the change here.

**Comment by bomanimc** [comment]
Hi @yonaymoris. Thanks for the update here. It looks like about 33 or so additional files were updated in your last PR due to what looks like linting changes. Was this intentional?

At the moment, it doesn't look like this project has a defined linting structure. Before linting across the project, it might be best to add in some configs so that other contributors can align with the style as well.

**Comment by yonaymoris** [comment]
No, I think I ran `yarn format` before pushing the changes, and that's what happened. 
Apologizing for a dumb question, would you mind guiding me on how to add in those configs?

**Comment by bomanimc** [comment]
Hi Yona. Apologies for the very delayed response here. I got caught up trying to find a new job and I forgot to follow up on this PR! Your contribution is very appreciated!

My suggestion is to focus this PR on the helpful updates to the Community page. To get this PR ready, let's start by removing the large number of lint-related changes here and focusing only on the additions that you included by hand. Then, if you're interested, we can talk about how you could create a separate PR that adds a great linting approach to this entire codebase for all contributors. How does that sound?

In case it's useful, I'll share how I'd approach removing the lint-related changes from the PR. You may already be familiar with these steps; adding the details here just in case it could be useful (to you or to someone who looks at this commit in the future). One approach you could take to fixing this is the following:
1. Change to the branch `yonaymoris:yona-communityp-fixes` with `git checkout yona-communityp-fixes`.
2. Run `git log`. Assuming you haven't made other changes to this branch, the latest commit (at the top) should be called "Remove the extra project".
3. Run `git reset HEAD~1`. This will tell Git to undo the last commit, but leave all of the file changes intact. This is similar to what the state of the repo would have looked like right before you ran `git commit -am ....` to make the commit we just reset. The `HEAD~1` is essentially like saying "go back to one commit behind the most recent commit`.
4. Run `git status`. You should now see all of the files that were changed in the commit we just undid. They should be unstaged (showing up as red in many terminals).
5. Run `git add <file path>` for the specific files you intended to change (e.g the files you updated in the docs/ directory, etc).
6. Run `git commit -m "<your new commit name">` to commit only the file that you staged with `git add` in the last step. Note that we want to avoid running `git commit -am` because the `-am` flag will automatically stage all of the other files we don't want to add.
7. Lastly, update this branch with `git push --force`. This force push is necessary because we change the commit history on this branch by resetting a commit and replacing it with a new one.

That's it! Let me know how it goes or if I can be helpful! I'll prioritize quick follow-ups here to reconcile for my delay in responding.

**Comment by bomanimc** [comment]
Apologies! This got cloned temporarily while I was updating the default branch from `master` to `main`. Looking forward to your updates here if you'd still like to edit this PR!

**Comment by yonaymoris** [comment]
Thanks, Bomani!
Working on another PR now!

**Comment by yonaymoris** [comment]
Hi Bomani. Sorry, I got caught up on the finals. I just pushed a new commit which should be fine with the updated links in the markdown. There is still the change in the hello.js file left from seems like even older commits, I checked it, and seems like it wrapped the last function into brackets for some reason (?) which is interesting.

**Review by bomanimc — APPROVED** [review]
Thanks for the updates here! I'll be merging and deploying these changes shortly!

**Comment by bomanimc** [comment]
This should now be live!

**Pull request merged**